elementary/monitor/api/alerts/alerts.py

```diff
@@ -140,26 +142,17 @@ def _get_suppressed_alerts(
     ) -> List[str]:
         suppressed_alerts = []
         current_time_utc = datetime.utcnow()
-        for alert in alerts.alerts:
-            unique_id = alert.unique_id
-            suppression_interval = alert.alert_suppression_interval
-            last_sent_time = (
-                datetime.fromisoformat(last_alert_sent_times.get(unique_id))
-                if last_alert_sent_times.get(unique_id)
-                else None
+        for alert in [*alerts.alerts, *alerts.malformed_alerts]:
```

Comment:
unified between alerts and malformed alerts (they had the same logic but different field extracting)